### PR TITLE
Accurate Times for different timezones

### PR DESCRIPTION
**Pull request by Camwyn**
🎫 https://central.tri.be/issues/76618

Create a new constant with the TZ info in it so we don't have to jump through so many hoops.

Also shifts around some duplicate-ish functions to prep for deptrecation.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Timezones.php:394](https://github.com/moderntribe/tribe-common/blob/aaa5ec4337c9daf53b91d1651d61687fbd51b71a/src/Tribe/Timezones.php#L394)
:warning: | Comment refers to a TODO task | [Timezones.php:433](https://github.com/moderntribe/tribe-common/blob/aaa5ec4337c9daf53b91d1651d61687fbd51b71a/src/Tribe/Timezones.php#L433)
:warning: | Comment refers to a TODO task | [Timezones.php:493](https://github.com/moderntribe/tribe-common/blob/aaa5ec4337c9daf53b91d1651d61687fbd51b71a/src/Tribe/Timezones.php#L493)
:x: | Must use &quot;self::&quot; for local static member reference | [Timezones.php:500](https://github.com/moderntribe/tribe-common/blob/aaa5ec4337c9daf53b91d1651d61687fbd51b71a/src/Tribe/Timezones.php#L500)
:warning: | Comment refers to a TODO task | [Timezones.php:536](https://github.com/moderntribe/tribe-common/blob/aaa5ec4337c9daf53b91d1651d61687fbd51b71a/src/Tribe/Timezones.php#L536)
:x: | Don't leave error_log lying around | [Timezones.php:549](https://github.com/moderntribe/tribe-common/blob/aaa5ec4337c9daf53b91d1651d61687fbd51b71a/src/Tribe/Timezones.php#L549)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Timezones.php [hunk 1]
```diff
@@ -540,5 +426,140 @@ public static function timezone_from_utc_offset( $utc_offset_string ) {
...
+	 *
+	 * @param string $unix_timestamp
+	 * @param string $tzstring
+	 *
+	 * @return string
+	 */
+	public static function adjust_timestamp( $unix_timestamp, $tzstring ) {
+		try {
+			$local = self::get_timezone( $tzstring );
+			error_log(
```
Comment (bordoni) [inline src/Tribe/Timezones.php]: Debug line.

File: src/Tribe/Timezones.php [hunk 2]
```diff
@@ -540,5 +426,140 @@ public static function timezone_from_utc_offset( $utc_offset_string ) {
...
+	 * @since  4.0.7
+	 *
+	 * @todo revise to eliminate all of these: maybe_get_tz_name, apply_offset, timezone_from_utc_offset, and adjust_timestamp
+	 *
+	 * @param  string|int|float $timezone The timezone
+	 *
+	 * @return string           The Guessed Timezone String
+	 */
+	public static function maybe_get_tz_name( $timezone ) {
+		if ( ! Tribe__Timezones::is_utc_offset( $timezone ) && ! is_numeric( $timezone ) ) {
```
Comment (bordoni) [inline src/Tribe/Timezones.php]: use `self`

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Timezones.php:394](https://github.com/moderntribe/tribe-common/blob/4dd86503298b59c239eee3d062302703206295db/src/Tribe/Timezones.php#L394)
:warning: | Comment refers to a TODO task | [Timezones.php:433](https://github.com/moderntribe/tribe-common/blob/4dd86503298b59c239eee3d062302703206295db/src/Tribe/Timezones.php#L433)
:warning: | Comment refers to a TODO task | [Timezones.php:493](https://github.com/moderntribe/tribe-common/blob/4dd86503298b59c239eee3d062302703206295db/src/Tribe/Timezones.php#L493)
:warning: | Comment refers to a TODO task | [Timezones.php:536](https://github.com/moderntribe/tribe-common/blob/4dd86503298b59c239eee3d062302703206295db/src/Tribe/Timezones.php#L536)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [Timezones.php:394](https://github.com/moderntribe/tribe-common/blob/b6f5028ad133da76662dc82be8ded77fd416939d/src/Tribe/Timezones.php#L394)
:warning: | Comment refers to a TODO task | [Timezones.php:433](https://github.com/moderntribe/tribe-common/blob/b6f5028ad133da76662dc82be8ded77fd416939d/src/Tribe/Timezones.php#L433)
:warning: | Comment refers to a TODO task | [Timezones.php:493](https://github.com/moderntribe/tribe-common/blob/b6f5028ad133da76662dc82be8ded77fd416939d/src/Tribe/Timezones.php#L493)
:warning: | Comment refers to a TODO task | [Timezones.php:536](https://github.com/moderntribe/tribe-common/blob/b6f5028ad133da76662dc82be8ded77fd416939d/src/Tribe/Timezones.php#L536)

> via jenkins/codesniffer-phpcs

**Pull request merged**